src/operator/optimizer_op-inl.h

```diff
@@ -212,24 +218,31 @@ struct RMSPropAlexParam : public dmlc::Parameter<RMSPropAlexParam> {
   float clip_gradient;
   float clip_weights;
   DMLC_DECLARE_PARAMETER(RMSPropAlexParam) {
-    DMLC_DECLARE_FIELD(lr).describe("learning_rate");
-    DMLC_DECLARE_FIELD(gamma1).set_default(0.95f).describe("gamma1");
-    DMLC_DECLARE_FIELD(gamma2).set_default(0.9f).describe("gamma2");
-    DMLC_DECLARE_FIELD(epsilon).set_default(1e-8f).describe("epsilon");
-    DMLC_DECLARE_FIELD(wd).set_default(0.0f).describe("weight decay");
+    DMLC_DECLARE_FIELD(lr)
+    .describe("Learning rate");
+    DMLC_DECLARE_FIELD(gamma1).set_default(0.95f)
+    .describe("Decay rate.");
+    DMLC_DECLARE_FIELD(gamma2).set_default(0.9f)
+    .describe("Decay rate.");
+    DMLC_DECLARE_FIELD(epsilon).set_default(1e-8f)
+    .describe("A small constant for numerical stability.");
+    DMLC_DECLARE_FIELD(wd).set_default(0.0f)
+    .describe("Weight decay. "
```

Comment:
Same as above:

Change to:

"Weight decay augments the objective function with a regularization term that penalizes large weights. The penalty scales with the square of the magnitude of each weight."